vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -181,7 +203,7 @@ func (r RefCountsMap) Init(d drivers.VolumeDriver, mountDir string, name string)
 	err = r.discoverAndSync(c, d)
```

Comment:
discoverAndSync purges existing refCnt map. I don't see any locking. What if mount/unmount requests comes in when we are in retryCalculate->calculate->discoverAndSync?